src/benchmarks/micro/corefx/System.Text.Json/Serializer/WriteJson.GenericCollection.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class ReadGenericCollection
+    {
+        private static string _jsonString;
+
+        [Params(2, 50, 100)]
+        public int ElementCount;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            ISet<string> _iset = new HashSet<string>();
+            for (int i = 0; i < ElementCount; i++)
+            {
+                _iset.Add($"hello{i}");
+            }
+
+            _jsonString = JsonSerializer.ToString(_iset);
+        }
+
+        [Benchmark]
+        public ISet<string> DeserializeISet()
```

Comment:
Please share the perf results of any new tests added within the PR you add them in.

Reply:
Looking at the Error/Std Dev of some of the results shared, I recommend commenting out the RecommendedConfig before running the tests locally to improve accuracy. 

For example, this is quite inaccurate:

DeserializeISet | 2 | 21,030.9 ns | 3,649.67 ns | 10,412.72 ns
-- | -- | -- | -- | --

https://github.com/dotnet/performance/blob/2e05afce5332d5e0a308e6208112b4205d3faf75/src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs#L17-L20